src/commands/api-mesh/status.js

```diff
@@ -44,94 +44,87 @@ class StatusCommand extends Command {
 			try {
 				const { showCloudflareURL: showEdgeMeshUrl } = await getTenantFeatures(imsOrgCode);
 				const mesh = await getMesh(imsOrgId, projectId, workspaceId, workspaceName, meshId);
-				if (showEdgeMeshUrl) {
-					const meshDeployments = await getMeshDeployments(
-						imsOrgCode,
-						projectId,
-						workspaceId,
-						meshId,
-					);
-					this.log(chalk.blackBright('Edge Mesh Status:'));
-					switch (String(meshDeployments.status).toLowerCase()) {
-						case 'success':
-							this.log(
-								'******************************************************************************************************',
-							);
-							this.log('Your mesh was successfully built.');
-							this.log(
-								'******************************************************************************************************',
-							);
-							break;
-						case 'provisioning':
-							this.log(
-								'******************************************************************************************************',
-							);
-							this.log('Your mesh is provisioning.');
-							this.log(
-								'******************************************************************************************************',
-							);
-							break;
-						case 'error':
-							this.log(
-								'******************************************************************************************************',
-							);
-							this.log('Your mesh encountered the following error: ', meshDeployments.error);
-							this.log(
-								'******************************************************************************************************',
-							);
-							break;
-						default:
-							//Case when deployments table has no status and only initial metadata
-							this.log(
-								'******************************************************************************************************',
-							);
-							this.log('Your mesh status is not available. Please wait for a while and try again.');
-							this.log(
-								'******************************************************************************************************',
-							);
-					}
-					this.log(chalk.blackBright('Legacy Mesh Status:'));
-				}
+				this.log(
+					'******************************************************************************************************',
+				);
 				switch (mesh.meshStatus) {
 					case 'success':
 						this.log(
-							'******************************************************************************************************',
-						);
-						this.log('Your mesh has been successfully built.');
-						this.log(
-							'******************************************************************************************************',
+							`${chalk.bold(`Legacy Mesh:`)} ${chalk.blackBright(`has been successfully built.`)}`,
```

Comment:
blackBright (grey) may provide less contrast on a dark theme'd terminal. Can we stick to the default log color for the messages? It will provide more contrast out of the box in both a light/dark background and be customizable based on the user's terminal theme instead of us being opinionated.

Reply:
updated in recent commit. Thanks